src/components/OperationWizard.vue

```diff
@@ -145,6 +179,26 @@ function goForward() {
         </v-row>
       </v-card-text>
     </v-slide-y-reverse-transition>
+    <v-row
```

Comment:
Line 171 should be where this goes - i.e. we should render an image picker when there is an input option of type `file`. Then you can remove the placeholder text I have there instead of the widget

Reply:
a fun challenge. I'll work on it